pandas/core/window/rolling.py

```diff
@@ -1414,8 +1454,14 @@ def skew(self, **kwargs):
     )
 
     def kurt(self, **kwargs):
+        window_func = self._get_cython_func_type("roll_kurt")
+        kwargs.pop("require_min_periods", None)
```

Comment:
what happens here if the user passes a weird value for require_min_periods?

Reply:
`require_min_periods` is effectively an internal variable and shouldn't be expected from an external API. I need to pop here because of kwargs passed from other `super` calls.